### ISSUE TITLE
"All facilities" breadcrumb naming not accessible

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4097]
## Area or Component
Facility form

## Quick Summary
The breadcrumb back to look at the list of facilities is labelled "all facilities", but this name is not accessible enough to screenreader users without the back arrow context (the arrow is aria-hidden.)

### How to Replicate
1. Go to settings page
2. Choose "manage facilities"
3. Click on a facility
4. Inspect the "all facilities" link

## Accessibility Violation
Name: The element's name is missing or incorrect: https://cdc-devtools.dequecloud.com/rule-help/en/aria-name-missing-incorrect

## Action Requested
Change the name of the link to something more descriptive (e.g. with an action), like "View all facilities" or "back to all facilities".

## Screenshots (optional)
<img width="982" alt="Screen Shot 2022-07-27 at 4 26 26 PM" src="https://user-images.githubusercontent.com/80282552/181389412-e36c4694-d01c-4de6-802c-0c0e6d9e5919.png">

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1346058064&range=8:8
